OutSystems/outsystems-datagrid-web
# GRID-327 - Blocking rowheader resize

Test [Page](https://pp-datagridteam-dev.outsystemsenterprise.com/Test_DataGridSample/Test_Grid_327a.aspx)

Analysis:
- Can not resize column if allowResize is set to false

Fixing:
- Set allowResize to true, change the width, return it to false

TestCase:
1. Try to resize autoNumber column
2. Scroll down to the end
3. Verify the adjust of this column’s width
4. Scroll up to the top
5. Verify the adjust of this column’s width

Expected:
- Items 3 and 5 should work and resize the column’s width to make value visible
